Update tableau.md (#4961)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/tableau.md

- Some functions in Analysis --> Create Calculated Field
- Filtering on a Date Part time dimension for a Cumulative metric type
- Changing your date dimension to use "Week Number"

> - Performing joins between tables that are created by the Semantic Layer. The Semantic Layer handles joins for you, so there's no need to join components together in the Semantic Layer (note: you _can_ join tables from Semantic Layer to ones outside in your data platform)

Reviewer: small tweak bc the 'can' needed a space after and simplifying lang
![Screenshot 2024-02-21 at 11 28 05](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/b30338a1-1656-4842-a49b-12ad1e289f02)

```suggestion
- Performing joins between tables that the dbt Semantic Layer creates. It handles joins for you, so there's no need to join components in the dbt Semantic Layer. Note, that you _can_  join tables from the dbt Semantic Layer to ones outside your data platform.
```
